test/unit/compas/CompasChangeSet.test.ts

```diff
@@ -0,0 +1,30 @@
+import { fixture, html, expect } from '@open-wc/testing';
+
+import "../../../src/compas/CompasChangeSet.js";
+import {CompasChangeSetRadiogroup} from "../../../src/compas/CompasChangeSet.js";
+
+describe('compas-changeset-radiogroup', () => {
+  let element: CompasChangeSetRadiogroup;
+  beforeEach(async () => {
+    element = <CompasChangeSetRadiogroup>(
+      await fixture(
+        html`<compas-changeset-radiogroup></compas-changeset-radiogroup>`
+      )
+    );
+  });
+
+  it('has 3 item entries', () => {
+    expect(element.shadowRoot!.querySelectorAll('mwc-list > mwc-radio-list-item'))
+      .to.have.length(3)
+  });
+
+  it('has the correct value for MAJOR Item', () => {
```

Comment:
Maybe also add the ones for MINOR and PATCH changeset? I know, changes are small that others are failing if this one succeeds, but it's a small change! :)

Reply:
Added tests, but it was more to see if the description was displayed.